modules/vector-search/pages/create-vector-search-index-rest-api.adoc

```diff
@@ -0,0 +1,59 @@
+= Create a Vector Search Index with the REST API and curl/HTTP
+:page-topic-type: guide
+:description: You can create a Vector Search index with the Search Service API and Couchbase Capella. 
+
+[abstract]
+{description}
+
+You must create a Search index before you can xref:run-vector-search-rest-api.adoc[run a search] with the Search Service.
+// Add link
+
+== Prerequisites 
+
+* You have the Search Service enabled on a node in your database. 
+
+* You have a bucket with scopes and collections in your database. 
+
+* You have documents in a keyspace inside that bucket that contain vector embeddings. 
```

Comment:
Nit: I wonder if the language of these prereq items should be reconciled more with the same ones in Quick Mode procedure.

Reply:
Good shout.